test/builder/pod.go

```diff
@@ -0,0 +1,115 @@
+package builder
+
+import (
+	"context"
+
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type PodBuilder struct {
+	kubeclient *kubernetes.Clientset
+	obj        *corev1.Pod
+}
+
+func NewPodBuilder(namespace, name string) *PodBuilder {
```

Comment:
Add method `AddSecurityContext` to let fluent pod run with `privileged: true`

Reply:
Well add as needed but wasn't required for this test. Not certain we will need in with functional tests in general